man/attributes.Rd

```diff
@@ -0,0 +1,9 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
Do we need these to be in an rda file? Why not check them into source and add to the package environ?

Reply:
I prefer the rda route (easier to update), but I have adjusted this by moving get_vaa_names() to nhdplusTools.R and adding the variable names within the function. Again though, keeping track of 40 variable names seems easier to read off the fst file and store as rda.